From cd9d741d476b60d95f9ff8df1492715b47cfc6c0 Mon Sep 17 00:00:00 2001
From: skyel <razvan.ghitulete@gmail.com>
Date: Tue, 17 Dec 2013 14:32:28 +0100
Subject: [PATCH 04/18] llvmlinux-lto: Remove -flto while compiling mod
 directory

This is necessary due to the kernel script checks for valid elf files.
Since we plan to generate elf files a bit later in the building phase the
scripts will not be able to get the appropriate system parameters from the
bitcode files, thus we disable lto here

Signed-off-by: skyel <razvan.ghitulete@gmail.com>
---
 scripts/mod/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/scripts/mod/Makefile b/scripts/mod/Makefile
index c11212f..bc6b0df 100644
--- a/scripts/mod/Makefile
+++ b/scripts/mod/Makefile
@@ -1,3 +1,5 @@
+KBUILD_CFLAGS += ${DISABLE_LTO}
+
 hostprogs-y	:= modpost mk_elfconfig
 always		:= $(hostprogs-y) empty.o
 
-- 
1.8.3.1

